### PR TITLE
Native_GetPlayerInfo - add bonus level

### PR DESCRIPTION
**Pull request by tslashd**
Added a return for which bonus the client is currently on.
Small fixes for g_iInBonus[client]

### CODE REVIEW

File: addons/sourcemod/scripting/surftimer/commands.sp [hunk 1]
```diff
@@ -1185,7 +1185,8 @@ public Action Command_ToBonus(int client, int args)
 		zoneGrp = 1;
 
 	g_bInBonus[client] = true;
-	g_iInBonus[client] = zoneGrp;
+	if (g_mapZoneGroupCount > zoneGrp)
```
Comment (tslashd) [inline addons/sourcemod/scripting/surftimer/commands.sp:1188]: This stops a bug where if you type `/b <anyNumber>` that number will be assigned to `g_iInBonus[client]`

Have not noticed it breaking something on live version, but now this variable does not get appended with a new number if there are not that many bonus zones.

File: addons/sourcemod/scripting/SurfTimer.sp [hunk 2]
```diff
@@ -1939,6 +1939,7 @@ public void OnClientConnected(int client)
 	g_Stage[0][client] = 1;
 	g_bWrcpTimeractivated[client] = false;
 	g_CurrentStage[client] = 1;
+	g_iInBonus[client] = 0;
```
Comment (tslashd) [inline addons/sourcemod/scripting/SurfTimer.sp:1942]: Was throwing errors in server console until client reaches a StartZone iirc as this value was being set after they touch a start zone

Comment (tslashd) [reply addons/sourcemod/scripting/SurfTimer.sp:1942]: image



**N I C E**

File: addons/sourcemod/scripting/surftimer/commands.sp (hunk 1 above)
Comment (tslashd) [reply addons/sourcemod/scripting/surftimer/commands.sp:1188]: image



**N I C E**

**Review by Bara — APPROVED** [review]

**Pull request merged**